pytorch_lightning/utilities/device_dtype_mixin.py

```diff
@@ -45,7 +45,7 @@ def device(self) -> Union[str, torch.device]:
 
         return device
 
-    def to(self, *args, **kwargs) -> Module:
+    def to(self, *args: Any, **kwargs: Any) -> 'DeviceDtypeModuleMixin':
```

Comment:
why is Module not good here?

Reply:
@awaelchli it's not specific enough. I.e. when having model here, the checkers don't know about dtype and device properties.